lua/lazyvim/plugins/lsp/init.lua

```diff
@@ -188,7 +192,18 @@ return {
         opts.capabilities or {}
       )
 
-      local function setup(server)
+      -- get all the servers that are available through mason-lspconfig
+      local have_mason, mlsp = pcall(require, "mason-lspconfig")
+      local all_mslp_servers = {}
+      if have_mason and vim.fn.has("nvim-0.11") == 1 then
+        all_mslp_servers = vim.tbl_keys(require("mason-lspconfig").get_mappings().lspconfig_to_package)
```

Comment:
As a quick fix for perf issues:

```suggestion
        all_mslp_servers = vim.tbl_keys(require("mason-lspconfig.mappings").get_mason_map().lspconfig_to_package)
```